src/main/java/it/unitn/disi/smatch/filters/SPSMMappingFilter.java

```diff
@@ -159,62 +223,113 @@ protected void filterMappingsOfChildren(INode sourceParent, INode targetParent,
      *
      * @param source           Source list of siblings.
      * @param target           Target list of siblings.
-     * @param semanticRelation a char representing the semantic relation as defined in IMappingElement.
+     * @param rscList          List of source siblings to be reordered.
```

Comment:
Having two names with 1-letter difference makes confusing them very easy. And often leads to hard to find bugs due to typos. Moreover, half of the name is "List" which is type and is mostly irrelevant in any modern IDE. Worth renaming.